src/twisted/conch/topfiles/8784.feature

```diff
@@ -0,0 +1 @@
+Add support in twisted.conch.ssh for hmac-sha2-384
```

Comment:
this should be a release notes message and not a commit message.

It can be phrases as

```
twisted.conch.ssh now supports hmac-sha2-384
```